xml/System/GCMemoryInfo.xml

```diff
@@ -313,9 +315,9 @@ The memory after `OBJ_D` is not considered part of the `FragmentedBytes` but is
         <ReturnType>System.ReadOnlySpan&lt;System.TimeSpan&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the pause durations. For blocking GCs, there is one pause; for background GC, there are two.</summary>
+        <summary>Gets the durations of the pauses when all managed threads are suspended.</summary>
         <value>A timespan representing the pause durations.</value>
-        <remarks>To be added.</remarks>
+        <remarks> For blocking GCs, there is one pause; for background GC, there are two (they occur when the ephemeral generations 0 and 1 are collected).</remarks>
```

Comment:
it actually doesn't include the ephemeral pauses at all. we just do an initial pause to figure out a bit of info like what gen to collect and etc. and later another pause.